relayer-engine/src/listener/eventHarness.ts

```diff
@@ -39,13 +40,18 @@ export async function consumeEventHarness(
   providers: Providers,
   extraData?: any[],
 ): Promise<void> {
+  const parsedVaa = parseVaaWithBytes(vaa);
```

Comment:
What happens now if this throws?

Reply:
I was thinking about that and whether this should ever throw, but I'll wrap this in a separate try.

Reply:
@JoeHowarth in reality, nothing happens. The vaa is being parsed outside of a `try` here: `consumeEventWithMissedVaaDetection`, and here:
```
  stream.on("data", (vaa: { vaaBytes: Buffer }) => {
  const parsed = wormholeSdk.parseVaa(vaa.vaaBytes);
```

The exception will bubble up but ultimate nobody is awaiting that promise. 

I'll add a catch somewhere else but if anything, we're parsing the vaa multiple times in each step (I could address that in a separate PR).

Reply:
Hmm, yeah we don't have to block on that here, but if nothing is awaiting it and it throws, I believe it takes out the process